Tests/Unit/Controller/TeaControllerTest.php

```diff
@@ -69,6 +81,7 @@ public function indexActionAssignsAllTeaAsTeasToView(): void
         $teas = $this->prophesize(QueryResultInterface::class)->reveal();
         $this->teaRepositoryProphecy->findAll()->willReturn($teas);
         $this->viewProphecy->assign('teas', $teas)->shouldBeCalled();
+        $this->responseFactoryProphecy->createResponse()->shouldBeCalled();
```

Comment:
Eventually add here a `->willReturn(new HtmlResponse())` or a proper prophesized response or setup response. Eventually do not prophecies the response factory if possibile (not looked into it)

Reply:
I didn't find a way not to prophesize this and still get the tests running However it feels like I am testing Core Details now and not really the Tea Controller...